apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/Adapter.kt

```diff
@@ -1,7 +1,7 @@
 package com.apollographql.apollo3.api
 
-import com.apollographql.apollo3.api.internal.json.BufferedSinkJsonWriter
-import com.apollographql.apollo3.api.internal.json.BufferedSourceJsonReader
+import com.apollographql.apollo3.api.json.BufferedSinkJsonWriter
+import com.apollographql.apollo3.api.json.BufferedSourceJsonReader
```

Comment:
Promoted to public because this is general purpos Json parsing that I feel can be useful